templates/projects/web/project.json

```diff
@@ -55,8 +55,7 @@
       "npm install",
       "bower install",<% if(!grunt){ %>
```

Comment:
The same as in: https://github.com/OmniSharp/generator-aspnet/pull/536/files?diff=split#r47272678